src/main/resources/templates/datewarehouse/etl-jobs-template.json.vpt

```diff
@@ -74,6 +74,22 @@
 							}]
 						}
 					},
+					{
+					    "PolicyName": "GlueStudioAccess",
+                        "PolicyDocument": {
+                            "Version": "2012-10-17",
+                            "Statement": [{
+                                "Effect": "Allow",
+                                "Action": [
+                                    "s3:GetObject"
+                                ],
+                                "Resource":[
+                                    "arn:aws:s3:::aws-glue-studio-transforms-510798373988-prod-us-east-1/gs_common.py",
```

Comment:
we can allow any transforms from this bucket without micromanaging the policy:
arn:aws:s3:::aws-glue-studio-transforms-510798373988-prod-us-east-1/*